app/models/profile.rb

```diff
@@ -0,0 +1,11 @@
+class Profile < ApplicationRecord
+    belongs_to :user
+
+    validates :name, presence: true
+    validates :address, presence: true
+    validates :phoneNumber, presence: true
+    validates :birthday, presence: true
+    validates :breedingExperience, presence: true
```

Comment:
この辺の書き方は好みや思想で分かれるのですが以下のような書き方でもいけます。
```suggestion
    with_options presence: true do
      validates :name
      validates :address
      validates :phoneNumber
      validates :birthday
      validates :breedingExperience
    end
```

Reply:
修正しました。